src/backend/controllers/NotesController.js

```diff
@@ -0,0 +1,109 @@
+import { Response } from "miragejs";
+import { requiresAuth } from "../utils/authUtils";
+
+/**
+ * All the routes related to Cart are present here.
+ * These are private routes.
+ * Client needs to add "authorization" header with JWT token in it to access it.
+ * */
+
+/**
+ * This handler handles getting items to user's cart.
+ * send GET Request at /api/user/cart
+ * */
+export const getNoteItemsHandler = function (schema, request) {
+  const user = requiresAuth.call(this, request);
+  if (!user) {
+    new Response(
+      404,
+      {},
+      {
+        errors: ["The email you entered is not Registered. Not Found error"],
+      }
+    );
+  }
+  // const userNotes = schema.users.findBy({ _id: user }).notes;
+  return new Response(200, {}, { notes: user.notes });
+};
+
+/**
+ * This handler handles adding items to user's cart.
+ * send POST Request at /api/user/cart
+ * body contains {product}
+ * */
+
+export const addItemToNotesHandler = function (schema, request) {
+  const user = requiresAuth.call(this, request);
+  if (user) {
+    const { video } = JSON.parse(request.requestBody);
+    if (user.notes.some((item) => item._id === video._id)) {
+     
+      return new Response(
+        409,
+        {},
+        {
+          errors: ["The video is already in your liked videos"],
+        }
```

Comment:
give proper error message to note controller

Reply:
Changes made